charts/prometheus/values.yaml

```diff
@@ -553,9 +546,3 @@ prometheus-rules:
             expr: chainlink_multi_node_states{state!="Alive"} > 0
             keep_firing_for: "15m"
             description: "EVM Node ({{ $labels.network }} chainId {{ $labels.chainId }}, container {{ $labels.container }}) is down"
-          To_many_gcp_logs:
-            enabled: false
-            for: "5m"
-            description: "Container {{ $labels.container_name }} in pod {{ $labels.pod_name }} located in namespace {{ $labels.namespace_name }} sends to many logs ({{ $labels.log }})"
-            # since stackdriver has a problem with obtaining Delta metrics and the metrics are not accurate, we experimentally found that dividing the result by 5 gives the most accurate results. Therefore, we divide the result by 5
-            expr: rate(stackdriver_k_8_s_container_logging_googleapis_com_log_entry_count[10m])/5 > 1000
```

Comment:
why was it deleted?

Reply:
Its a project specific alert so i moved it to the projects that need this alert. Another reason is that we will use our own logging soon so this alert will be obsolete

Reply:
discussed internally - the decision is to keep gcp logs alert and remove pubsub alert